docs/en/token_endpoint.rst

```diff
@@ -225,6 +225,11 @@ ID Token
 ++++++++
 
 The ID Token is a JSON Web Token (JWT) that contains information on the user that has executed the authentication. The RPs MUST validate the ID Token.
+
+.. admonition:: |cieid-icon|
+
+  The RP MAY require the ID Token to be encrypted (see the parameter *id_token_encrypted_response_alg* in the :ref:`RP Metadata <MetadataRP>`). In this case the ID Token will be a **nested signed and encrypted JWT** and MUST contain the *cty* (Content-Type) parameter in the JOSE header with the value *JWT* (see :rfc:`7515#section-4`).
```

Comment:
If the RP exposes the parameters ... *id_token_encrypted_response_alg* in the :ref:`RP Metadata <MetadataRP>` ... the OP MUST encrypt the ID Token as a **nested signed and encrypted JWT**.

Reply:
fixed in https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/8ec33eca96a7075ce9c222eb16e07b2fc6820176